infrastructures/kubernetes/src/test/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesInternalRuntimeTest.java

```diff
@@ -378,7 +378,7 @@ public void throwsInfrastructureExceptionWhenErrorOccursAndCleanupFailed() throw
     doNothing().doThrow(InfrastructureException.class).when(namespace).cleanUp();
     when(k8sEnv.getServices()).thenReturn(singletonMap("testService", mock(Service.class)));
     when(services.create(any())).thenThrow(new InfrastructureException("service creation failed"));
-    doThrow(InfrastructureException.class).when(namespace).services();
```

Comment:
Can you elaborate on why we need this change? I don't see any connection with the purpose of the PR.